developer-extension/src/panel/app.tsx

```diff
@@ -14,9 +18,26 @@ export function App() {
       }}
       withGlobalStyles
     >
-      <Suspense fallback={<></>}>
-        <Panel />
-      </Suspense>
+      <Suspense fallback={<></>}>{isDisconnected ? <DisconnectAlert /> : <Panel />}</Suspense>
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** nice!